### PR TITLE
Allow overriding visual look in server settings

### PR DESCRIPTION
**Pull request by lubomir**
Two new keys can be specified in the server settings JSON file.

 * logo - a path to image file that should be added to header
 * customStyle - a path to stylesheet that should be included on the page

There should be no change in how the UI looks, but it should allow us to customize it to match header of our PDC instance. See `redhat-style` branch in my fork for how that could be done.

### CODE REVIEW

File: src/app.jsx [hunk 1]
```diff
@@ -405,31 +407,43 @@ module.exports = React.createClass({
         'hidden': !this.state.busy,
         'global-spin': !this.state.showresult
       });
+      var logo = null;
+      var style = null;
+      if (localStorage['logo']) {
+          logo = <img src='/redhat-logo.png' className="brand-logo" />;
```
Comment (hluk) [inline src/app.jsx]: This shouldn't contain `redhat`.

**Review by hluk — CHANGES_REQUESTED** [review]

**Comment by hluk** [comment]
Will the stylesheet and logo be part of internal ansible repos?

**Comment by zhutaoredhat** [comment]
http://www.patternfly.org/pattern-library/navigation/vertical-navigation/#/design
http://www.patternfly.org/pattern-library/navigation/horizontal-navigation/
@hluk  We could use the style of PatternFly navigation.

**Comment by lubomir** [comment]
Changes since previous version:
* by default the header looks the same as what PDC server itself does (using Patternfly)
* it's possible to customize the header with new options in server settings file (internal branding will live in ansible repo, PR is in progress)

File: src/app.jsx [hunk 2]
```diff
@@ -162,6 +166,9 @@ module.exports = React.createClass({
     });
     $.getJSON("serversetting.json", function( data ) {
       localStorage.setItem('server', data['server']);
+      self.state.logo = data['logo'] || null;
+      self.state.customStyle = data['customStyle'] || null;
+      self.state.links = data.links;
```
Comment (hluk) [inline src/app.jsx:187]: Can you also add some defaults or just comments for the options in serversettings.json?

Comment (lubomir) [reply src/app.jsx:187]: Good point. I can't add comments to the default JSON file. But documenting it in README might work as well.

Comment (hluk) [reply src/app.jsx:187]: > I can't add comments to the default JSON file.

Oh, yeah, I forgot how much JSON sucks :).

**Comment by hluk** [comment]
Sorry, I just pushed some conflicting code to master.

**Review by hluk — APPROVED** [review]

**Pull request merged**